docs/functions/json.md

```diff
@@ -0,0 +1,93 @@
+Following functions help you handle JSON documents.
+
+!!! important "The document object given to those functions could be of any type as it is always serialize as JSON string first"
```

Comment:
I think this is not enough clear

Reply:
Add : "json string can not be null or empty"
All functions accept an input parameter `json` of any type.